packages/terra-paginator/src/_paginationUtils.jsx

```diff
@@ -1,3 +1,5 @@
+import KeyCode from 'keycode-js';
```

Comment:
Let's drop the KeyCode import and export from this file and instead import it directly where needed.